pkg/scheduler/api/job_info.go

```diff
@@ -574,7 +574,7 @@ func (ji *JobInfo) Clone() *JobInfo {
 		PodGroup: ji.PodGroup.Clone(),
 
 		TaskStatusIndex:       map[TaskStatus]tasksMap{},
-		TaskMinAvailable:      ji.TaskMinAvailable,
+		TaskMinAvailable:      map[TaskID]int32{},
```

Comment:
Thanks for your report and bug fix. It's OK for me. A small tip is that making use of `make(map[TaskID]int32{})` instead of map directly maybe better. Anyway, it's not something matters.

Reply:
I've changed to make...

Reply:
A more graceful change is making map with cap ^_^.